src/layout.rs

```diff
@@ -5,24 +5,22 @@ use leptos_router::{Outlet, RoutingProgress};
 
 #[component]
 pub fn Layout(is_routing: ReadSignal<bool>) -> impl IntoView {
+  let ui_theme = expect_context::<RwSignal<String>>();
+
   view! {
-    <Suspense>
-      <ErrorBoundary fallback=|_| view! { Error! }>
-        <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
-        <Stylesheet id="leptos" href="/pkg/lemmy-ui-leptos.css"/>
-        <Link rel="shortcut icon" type_="image/ico" href="/favicon.ico"/>
-        <Meta name="description" content="Lemmy-UI-Leptos."/>
-        <Meta name="viewport" content="viewport-fit=cover"/>
-        <Script src="//cdn.jsdelivr.net/npm/eruda"/>
-        <Script>eruda.init();</Script>
-        <Title text="Brand from env"/>
+    <RoutingProgress is_routing max_time=std::time::Duration::from_millis(250)/>
```

Comment:
the SSR Mode of the page (Route) decides how many of the <Suspense> areas that get awaited or rendered at all.

if we default to PartiallyBlocked we have very limited (1?) async blocks that will be rendered. having one at the top level does not actually bound any specific async anyway so i think it is not functional.

testing things with javascript disabled you get a very different view of the site. SSR only mode is actually very limited in its abilities.

i guess having a catch all error boundary at the top level is somewhat useful for the user in the event of a completely unhandled app error, although it would be nice to catch everything at a much lower level and still provide some UI.